mimic/test/test_core.py

```diff
@@ -55,3 +58,47 @@ def test_from_plugin_includes_all_plugins(self):
             len(plugin_apis),
             len(list(core.entries_for_tenant('any_tenant', {},
                                              'http://mimic'))))
+
+    def test_load_domain_plugin_includes_all_domain_plugins(self):
+        """
+        Using the :func:`MimicRoot.fromPlugin` creator for a
+        :class:`MimicCore`, domain mocks implementing `class`:`IAPIDomainMock`
+        are included.
+        """
+        # build a fake plugins package with a domain plugin
+        self.root = FilePath(self.mktemp())
+        self.root.createDirectory()
+        self.package = self.root.child('fakeplugins')
+        self.package.createDirectory()
+        init = b"""from fakeplugins._domain import domain_plugin
+__all__ = [domain_plugin]
+"""
+        self.package.child('__init__.py').setContent(init)
```

Comment:
This clean-up should consist of `sys.modules.pop("mimic.plugins.fake_plugin")` and `del mimic.plugins.fake_plugin`.